packages/upswyng-web/src/components/Resource.tsx

```diff
@@ -167,6 +168,9 @@ export const Resource = () => {
           longitude={resource.longitude}
         />
       )}
+      <RouterLink to={`/report-issue/${resourceId}`} style={{ color: "white" }}>
+        Report an Issue
+      </RouterLink>
```

Comment:
For this link, let's also use MUI's link component. You can use a combo by:

`import { Link as MuiLink } from "@material-ui/core/Link"`

and then using its component prop to combine the two:

`<MuiLink component={RouterLink} to={`/report-issue/${resourceId}`}>...</MuiLink>`

Reply:
On 2nd thought, lets make this appear as a [button with an icon](https://material-ui.com/components/buttons/#buttons-with-icons-and-label) and use the report a problem icon:

```jsx
import ReportProblemIcon from '@material-ui/icons/ReportProblem
...
<Button
   component={RouterLink}
   variant="contained"
   color="secondary"
   startIcon={<ReportProblemIcon />}
   to={`/report-issue/${resourceId}`}
>
   Report a Problem
</Button>
```

Sorry I know this didn't have a proper mockup 😖

Reply:
![Screen Shot 2020-03-12 at 5 05 02 PM](https://user-images.githubusercontent.com/51095001/76575017-a16d2700-6483-11ea-86ff-9d82dce9d959.png)

How does this look?

Reply:
Is it too much?

Reply:
![Screen Shot 2020-03-12 at 5 06 28 PM](https://user-images.githubusercontent.com/51095001/76575088-d37e8900-6483-11ea-87af-8d6f2e4b65fa.png)
This is without style variants.